### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kathrynle20**
I added the complimentary Pandas Profiling link, installation line and a small description.

### CODE REVIEW

**Comment by amueller** [comment]
Please use "Fix #issueNumber" in your PR description (and you can do it more than once). This way the associated issue gets closed automatically when the PR is merged. For more details, look at [this](https://github.com/blog/1506-closing-issues-via-pull-requests).

I think the description is good but it would be ideal to focus a bit more on the difference to dabl. I'd say pandas profiling is more geared towards detailed univariate statistics, while dabl is more geared towards supervised learning.

**Comment by kathrynle20** [comment]
This is Fix #213.
I added the Pandas Profiling link, installation line, and a small description. Pandas Profiling is different from dabl in that it focuses on univariate statistics and presenting a full data report whereas dabl is focused on supervised learning and provides an easier way to clean up and analyze data.

**Comment by kathrynle20** [comment]
Sorry, I meant to write Fix #178.

File: README.md [hunk 1]
```diff
@@ -28,3 +28,14 @@ over different gradient boosting models and other tree ensembles and
 potentially kernel methods.
 
 Stay Tuned!
+
+## Pandas Profiling package
+The Pandas Profiling package is useful for initial data analysis. Using Pandas Profiling can provide a thorough summary of the data in only a single line of code. Using the ```ProfileReport()``` method, you are able to access a HTML reoprt of your data that can help you make initial correlations and identity missing data.
```
Comment (amueller) [inline README.md]:
```suggestion
The Pandas Profiling package is useful for initial data analysis. Using Pandas Profiling can provide a thorough summary of the data in only a single line of code. Using the ```ProfileReport()``` method, you are able to access a HTML report of your data that can help you find correlations and identify missing data.
```

**Pull request merged**